test/test.js

```diff
@@ -26,7 +25,13 @@ describe('Routes', () => {
   before(async () => {
     client = new pg.Client({ connectionString: DATABASE_URL })
     await client.connect()
-    const handler = await createHandler({ client, logger: noopLogger })
+    const handler = await createHandler({
+      client,
+      logger: {
+        info () {},
+        error (...args) { console.error(...args) }
```

Comment:
I found an issue with `noop-logger`: Ignoring all logs was bad, because if something errored badly you wouldn't see a needed log line in the test output. This changes it so that only expected log lines are hidden